docs/preprocess.md

```diff
@@ -0,0 +1,30 @@
+# Preprocess
+
+`@sveltejs/vite-plugin-svelte/preprocess` exports utilities to preprocess Svelte components using Vite's built-in transformers.
+
+## vitePreprocess
+
+- **Type:** `{ script?: boolean, style?: boolean | InlineConfig | ResolvedConfig }`
+- **Default:** `{ script: true, style: true }`
+
+  A Svelte preprocessor that supports transforming TypeScript, PostCSS, SCSS, Less, Stylus, and SugarSS. These are transformed when the script or style tags have the respective `lang` attribute.
```

Comment:
```suggestion
A Svelte preprocessor that supports transforming TypeScript, PostCSS, SCSS, Less, Stylus, and SugarSS. These are transformed when the script or style tags have the respective `lang` attribute.
```

Reply:
The indent is intentional with the `config.md` convention